packages/mermaid/src/docs/package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "docs",
```

Comment:
This isn't something for you to do in this PR, but what do you think about the `packages/mermaid/src/docs` folder into something like `packages/docs`? It might make it easier to split out what `devDependencies` we have for `mermaid`, and what `devDependencies` we only need for the docs.

Reply:
Yes, that can be done.

Reply:
Thanks for making an issue, (link at https://github.com/mermaid-js/mermaid/issues/4382, in case anyone is looking at this PR from the future)!